api/registry/registry_integration.go

```diff
@@ -35,14 +35,34 @@ func (d *idm) Init(ctx types.Context, config gofig.Config) error {
 	d.config = config
 	d.ctx = ctx
 	d.used = map[string]int{}
-	ctx.WithField("pathCache", "").Debug("checking volume path cache setting")
+	ctx.WithField(types.ConfigIntegrationVolPathCache, d.pathCache()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"empty path for unused volume setting")
+	ctx.WithField(types.ConfigIntegrationVolUnmountIgnoreUsed,
+		d.ignoreUsedCount()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"ignore used count on unmount setting")
+	ctx.WithField(types.ConfigIntegrationVolMountPreempt, d.preempt()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"pre-emptive mount setting")
+	ctx.WithField(types.ConfigIntegrationVolCreateDisable, d.disableCreate()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"volume creation disabled setting")
+	ctx.WithField(types.ConfigIntegrationVolRemoveDisable, d.disableRemove()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"volume removal disabled setting")
+
```

Comment:
@akutz Having a hard time getting the ServiceName() to show up where it should. I am assuming this should be in the client.go file?

```
	if config.IsSet(types.ConfigService) {
		c.ctx = c.ctx.WithServiceName(config.GetString(types.ConfigService))
	}
```

Even with this set, it still doesn't seem to be making it down to the drivers. Ideas?